### PR TITLE
Adding unit tests for JEEventInternal to demonstrate changing from AnyObject to Any does not have deleterious impact, also reformed some other instances of AnyObject to Any.

### PR DESCRIPTION
**Pull request by richardrauser-je**
This PR also includes addition of cocoapods testspec for the JustTrack pod to support these unit tests.

### CODE REVIEW

**Review by shabeerhussain-je — APPROVED** [review]

**Comment by ghost** [comment]
Thanks @richardrauser-je for adding the testspec. Can you please move the files from `Example/Tests` in the testspec too (inside `JustTrack/UnitTests`? So that `TrackerTests.swift` can be run with the testspec and we have all the unit tests in one place.
Once done, we can also delete the `JustTrack_Tests` (now named `JustTrack_Example_Tests`) target from the demo app and the directive for it in the `Podfile`.
We have other modules already following this pattern w/ the testspecs and it turns out we don't need a test target in the demo app anymore. cc @shabeerhussain-je

**Comment by richardrauser-je** [comment]
@albertodebortoli-je Good thinking.. done!

File: JustTrack/UnitTests/JEEventInternalTests.swift [hunk 1]
```diff
@@ -0,0 +1,38 @@
+//
+//  JEEventInternalTests.swift
+//  JustTrack
+//
+//  Created by Richard Rauser [Contractor] on 01/10/2018.
```
Comment (joakimlazakis-je) [inline JustTrack/UnitTests/JEEventInternalTests.swift]: Can we change this to match the rest of _JustTrack_? Especially the _Contractor_ bit? :) @albertodebortoli-je @richardrauser-je

**Review by joakimlazakis-je — APPROVED** [review]

File: JustTrack/UnitTests/JEEventInternalTests.swift (hunk 1 above)
Comment (ghost) [reply JustTrack/UnitTests/JEEventInternalTests.swift]: @joakimlazakis-je thanks for noticing, I'll remove it.

Comment (richardrauser-je) [reply JustTrack/UnitTests/JEEventInternalTests.swift]: Ha, of course. :D

**Pull request merged**